pipeline/stage03_trigger_detection/configs/config_LENS.yaml

```diff
@@ -61,7 +61,10 @@ TRANSITION_PHASE: -1.570796
 # BLOCK - Minima
 ################
 # Number of neighbouring points which should be compared to define a local minimum.
-MINIMA_ORDER: 3
+INTERPOLATION: 0 # 0 if minima are detected through parabolic interpolation
```

Comment:
Variable names should indicate both their use and their type.
Thus `USE_QUADRATIC_INTERPOLATION` would be a better choice here, and should be a bool and not an int.